exporters/otlp/otlphttp/driver.go

```diff
@@ -45,24 +44,6 @@ import (
 const contentTypeProto = "application/x-protobuf"
 const contentTypeJSON = "application/json"
 
-// Keep it in sync with golang's DefaultTransport from net/http! We
-// have our own copy to avoid handling a situation where the
-// DefaultTransport is overwritten with some different implementation
-// of http.RoundTripper or it's modified by other package.
-var ourTransport *http.Transport = &http.Transport{
-	Proxy: http.ProxyFromEnvironment,
-	DialContext: (&net.Dialer{
-		Timeout:   30 * time.Second,
-		KeepAlive: 30 * time.Second,
-		DualStack: true,
-	}).DialContext,
-	ForceAttemptHTTP2:     true,
-	MaxIdleConns:          100,
-	IdleConnTimeout:       90 * time.Second,
-	TLSHandshakeTimeout:   10 * time.Second,
-	ExpectContinueTimeout: 1 * time.Second,
-}
-
```

Comment:
This shouldn't be removed, we still have the same concerns.  Use it where `http.DefaultTransport` is used.

Reply:
I'm not sure I understand why it might be a problem to use an overridden `http.DefaultTransport`.

Reply:
`http.DefaultTransport` is a single instance, and which can be and is commonly overridden.  This means we have a high likelihood of someone instrumenting that transport and sending us into a loop.

Reply:
Possible, but that's only one of the interesting ways a user may want to override the default transport and it would be nice to not require them to explicitly provide their new default to us as well to get us to honor it.  https://github.com/open-telemetry/opentelemetry-specification/issues/530 should provide a way forward for dealing with the export loop issue.  If we have to keep this cloned default transport here for now, I'd want to include a TODO and reference to that issue as a reminder to revisit it when we can.

Reply:
Another reason why relying on `http.DefaultTransport` may be bad is security concerns. Some nasty packages may intentionally change it in its `init` function e.g. to sniff the HTTP traffic.

Unfortunately, such vulnerabilities are common for such global configuration options...

IMO it should be at least documented that `otlphttp` uses `http.DefaultTransport` by default.